docs/_embed/partner_api.md

```diff
@@ -154,6 +158,30 @@ Get apps that can be integrated with my app.
 curl -L "https://api.zapier.com/v1/apps?client_id=${client_id}"
 ```
 
+Get a list of apps related to google
+
+```bash
+curl -L "https://api.zapier.com/v1/apps?client_id=${client_id}&category=google"
+```
+
+Get a list of apps that are included in the same Zap Template as the Google Calendar app
+
+```bash
+curl -L "https://api.zapier.com/v1/apps?client_id=${client_id}&is_in_zap_template_with=google-calendar"
+```
+
+Get the Google Calendar app
+
+```bash
+curl -L "https://api.zapier.com/v1/apps?client_id=${client_id}&title_search=google%calendar"
```

Comment:
I think we want either

```suggestion
curl -L "https://api.zapier.com/v1/apps?client_id=${client_id}&title_search=google+calendar"
```

or `title_search=google%20calendar` or `title_search=google calendar`.

Reply:
It doesn't look like this was updated either